tools/build_third_party_libs.py

```diff
@@ -209,13 +215,13 @@
         cmd = ["cmake", "--build", ".", "--config", "Release", "-j", f"{args.num_parallel_jobs}"]
 
     elif sys.platform == "linux":
-
         cmd = [
```

Comment:
Grouping `CMAKE` vars and `YAML_CPP` vars:

```
        cmd = [
            "cmake",
            "-DCMAKE_CXX_COMPILER=" + cxx_compiler,
            "-DCMAKE_CXX_FLAGS=" + cxx_flags,
            "-DCMAKE_POSITION_INDEPENDENT_CODE=ON",
            "-DCMAKE_VERBOSE_MAKEFILE=" + verbose_makefile,
            "-DYAML_CPP_BUILD_TESTS=OFF",
            os.path.join("..", "..")]
```

Reply:
I like (a) too. I've updated code to make it consistent across platforms.